### PR TITLE
[ADD] module account_balance_line

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by eLBati** [comment]
:+1:

**Comment by yvaucher** [comment]
Rebased to see what travis says

File: account_balance_line/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import account_move_line  # noqa
```
Comment (bwrsandman) [inline account_balance_line/__init__.py]: `# noqa` is uncessesary here and I would advise against using it.

File: account_balance_line/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.osv import osv, fields
+
+
+class account_move_line(osv.Model):
```
Comment (bwrsandman) [inline account_balance_line/account_move_line.py]: `osv` is deprecated, please use `orm`

**Comment by yvaucher** [comment]
@bwrsandman thanks for the reviews fixes done

**Comment by bwrsandman** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
:+1:

**Pull request merged**